### ISSUE TITLE
[ Windows] Tests are Failing on Windows.

### ISSUE THREAD

**Issue by canewsin** [Anshorei/zeronet_cryptography#1]
Due to this, zerunet cannot verify the sender on windows.

**Comment by canewsin** [comment]
I checked on wsl, all tests are passing, failing only happens on windows.

Windows Log : 
```
C:\Users\PramUkesh\Documents\RustProjs\zeronet_cryptography>cargo test
    Finished test [unoptimized + debuginfo] target(s) in 0.07s
     Running unittests (target\debug\deps\zeronet_cryptography-5dc1993cfc22c5d1.exe)

running 4 tests
test tests::test_msg_hash ... ok
test tests::test_verification ... FAILED
test tests::test_signing ... FAILED
test tests::test_creating ... ok

failures:

---- tests::test_verification stdout ----
thread 'tests::test_verification' panicked at 'assertion failed: `(left == right)`
  left: `false`,
 right: `true`', src\lib.rs:160:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- tests::test_signing stdout ----
thread 'tests::test_signing' panicked at 'assertion failed: `(left == right)`
  left: `false`,
 right: `true`', src\lib.rs:171:5


failures:
    tests::test_signing
    tests::test_verification

test result: FAILED. 2 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.06s

error: test failed, to rerun pass '--lib'
```

**Comment by canewsin** [comment]
Ok, nevermind tests are passing now, there is an issue while copying signature to my port.

**Comment by canewsin** [comment]
But Doc tests are failing so https://github.com/Anshorei/zeronet_cryptography/pull/2 merging this will solve the issue.

**Comment by Anshorei** [comment]
Merged #2
